SimplenoteUITests/SimplenoteUISmokeTestsSettings.swift

```diff
@@ -0,0 +1,56 @@
+import XCTest
+
+class SimplenoteUISmokeTestsSettings: XCTestCase {
+
+    override class func setUp() {
+        app.launch()
+        getToAllNotes()
+        _ = attemptLogOut()
+        EmailLogin.open()
+        EmailLogin.logIn(email: testDataExistingEmail, password: testDataExistingPassword)
```

Comment:
Side note. We could use the same value type approach from #1239 for the credentials.

```swift
struct TestUser { // or TestUserData, or UserData, or Credentials, or...
  let email: String
  let password: String
}
```

Reply:
Yes, this was my observation too, yesterday. Should be a part of refactor PR. Thank you!